acme_diags/derivations/acme.py

```diff
@@ -519,7 +519,8 @@ def cosp_histogram_standardize(cld):
     ]),
     'NET_FLUX_SRF': OrderedDict([
         (('FSNS','FLNS','LHFLX','SHFLX'), lambda fsns, flns, lhflx, shflx: netflux4(fsns, flns, lhflx, shflx)),
-        (('rsds','rsus','rlds','rlus','hfls','hfss'), lambda rsds, rsus, rlds, rlus, hfls, hfss: netflux6(rsds, rsus, rlds, rlus, hfls, hfss))
+        (('rsds','rsus','rlds','rlus','hfls','hfss'), lambda rsds, rsus, rlds, rlus, hfls, hfss: netflux6(rsds, rsus, rlds, rlus, hfls, hfss)),
+        (('QFLX'), lambda qflx: qflxconvert_units(qflx))
```

Comment:
@chengzhuzhang Let me know what the correct conversion function should be.

Reply:
`QFLX` is surface water flux with unit ["kg/m2/s"]
`LHFLX`is surface latent heat flux with unit ["W/m2"]
LHFLX = QFLX * 2.5 x 10^6     where 2.5x10^6 is latent heat of condensation/vaporization[J/kg]

Reply:
@chengzhuzhang I added this formula to the `LHFLX` derivation below. I don't see how it makes sense units-wise to put `QFLX` here at all.